### ISSUE TITLE
docker run ... ./interpolate build error: cannot create directory / Permission denied

### ISSUE THREAD

**Issue by arne-cl** [pelias/interpolation#217]
Running the ``docker run ... ./interpolate build`` from the README isn't possible (even after applying the fix from pelias/interpolation#216), as the `pelias` user doesn't have the necessary permissions.

```
docker run -i \
  -v /root/arne/pelias/interpolation-docker-dir/data:/data \
  -e 'BUILDDIR=/data/builddir' \
  -e 'POLYLINE_FILE=/data/polyline/eu-wide.osm.0sv' \
  -e 'OAPATH=/data/openaddresses' \
  -e 'PBF2JSON_FILE=/data/openstreetmap/eu-wide.osm.pbf' \
  -e 'PBF2JSON_BIN=/data/pbf2json.linux-x64' \
  pelias/interpolation ./interpolate build

mkdir: cannot create directory ‘/data/builddir/tmp’: Permission denied
```

As an **unsafe workaround**, one can remove the ``USER pelias`` line from the ``Dockerfile``.
I will look into the underlying issue (likely in the base image ``pelias/libpostal_baseimage``) next.

**Comment by arne-cl** [comment]
With the help of @orangejulius in pelias/docker-baseimage#14, I figured out that the error occurred
because the directory that I tried to mount as the `/data` volume was owned by root.

The `pelias/interpolation` container runs as the user `pelias` (i.e. with `UID:GID 1000:1000`),
so it can only write to the `/data` volume if it is mounted from a directory that is writable by the user with UID 1000 on the host system.

In other words: if you run as the "normal" (first non-root) user on a system, you should not run in this issue. If you see this error, run `id` to see the UID/GID of your current user and `ls -n /path/to/directory` to see the UID/GID of the directory you want to mount.
